lib/galaxy/model/__init__.py

```diff
@@ -2972,6 +2974,43 @@ class GenomeIndexToolData(Base, RepresentById):  # TODO: params arg is lost
     user: Mapped[Optional["User"]] = relationship()
 
 
+class ChatExchange(Base, RepresentById):
+
+    __tablename__ = "chat_exchange"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    user_id: Mapped[int] = mapped_column(ForeignKey("galaxy_user.id"), index=True, nullable=False)
+    job_id: Mapped[Optional[int]] = mapped_column(ForeignKey("job.id"), index=True, nullable=True)
```

Comment:
`nullable=True`: so a chat-exchange may or may not have an associated job?

Reply:
Yes, correct.  Currently job-based are the only way to create/interact, but this way we don't have to adjust the schema as functionality changes.